tsa-core/src/main/kotlin/org/usvm/machine/interpreter/TvmInputDict.kt

```diff
@@ -0,0 +1,456 @@
+package org.usvm.machine.interpreter
+
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.PersistentSet
+import kotlinx.collections.immutable.persistentHashSetOf
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.collections.immutable.toPersistentList
+import org.usvm.UBoolExpr
+import org.usvm.UExpr
+import org.usvm.machine.TvmContext
+import org.usvm.machine.splitHeadTail
+
+typealias KeySort = TvmContext.TvmCellDataSort
+typealias K = UExpr<KeySort>
+typealias ConstraintSet = PersistentList<UBoolExpr>
```

Comment:
If this is set, maybe it should be `PersistentSet`?

Also, I am not sure if this type alias simplifies the code